ffi/wasm/example/main.mjs

```diff
@@ -7,3 +7,12 @@ decimal.negate();
 const outWritable = new BufferWritable();
 decimal.write_to(outWritable);
 console.log(outWritable.getString());
+
+const dataProvider = new StaticDataProvider();
+
+const locale = new Locale("en");
```

Comment:
I'd pick Bengali (`bn`) to match the other tests, because it uses a 3-2-2-... separator scheme and isn't ASCII based.

test.c (and test.cpp in [my latest PR](https://github.com/unicode-org/icu4x/pull/768)) have some tests you can copy.